### ISSUE TITLE
Fix for Sierra installers

### ISSUE THREAD

**Issue by BEENNath58** [otya128/winevdm#1207]
Many Sierra installers have issues running with OTVDM. Some examples can be Power Chess 96, Leisure Suit Larry 7, 3D Ultra Pinball Creep Night, Rama, etc.

Although modern 32-bit installers exist for some popular games, they don't exist for the others. So, I believe it could be good if this bug can be fixed.

**Comment by cracyc** [comment]
The sierra.ini file needs to be moved from winevdm windows dir to the machine windows dir.  Winevdm can't predict whether the installed program is a win16 one or win32 one so I don't know of a sane way to make this work automatically.

**Comment by ks202005** [comment]
3D Ultra Pinball also has this issue. 

> The sierra.ini file needs to be moved from winevdm windows dir to the machine windows dir. 

What do you mean exactly? In my case, I'm running the SETUP.EXE straight from a mounted ISO disc image.

**Comment by cracyc** [comment]
What is the problem you are seeing?  If it's the hang on the cdrom test, just skip the machine tests when it asks.  If the game shows an error after install move the sierra.ini file from the <winevdmdir>\windows directory to the c:\windows directory.

**Comment by BEENNath58** [comment]
Hi. There are different types of problems actually. Most of the installers start and don't display on the screen. An example is Power Chess 96, the sound will play, and the icon appears on Taskbar but there's no window. However, the newer Power Chess 98 setup displays and runs properly.

**Comment by cracyc** [comment]
I tried 3d pinball creep and lsl7 and the screen appears for both.  They also install except the cdrom test fails (can be skipped) and sierra.ini is in the wrong place.

**Comment by BEENNath58** [comment]
> I tried 3d pinball creep and lsl7 and the screen appears for both. They also install except the cdrom test fails (can be skipped) and sierra.ini is in the wrong place.

3D Ultra Pinball Creep Night isn't still starting. I used a different machine, and the results are same as well. You can check the video:

https://user-images.githubusercontent.com/64649734/196353101-3d116ebd-c809-4707-a9dc-535e841df4c7.mp4

**Comment by BEENNath58** [comment]
I am aware of the SIERRA.INI transfer to run the game and the divide by 0 error in CD-ROM check, but they aren't the problems. This the actual problem I am seeing:

https://user-images.githubusercontent.com/64649734/196356096-b6fe9d76-298c-410a-8ca7-8e6170f46d73.mp4

**Comment by BEENNath58** [comment]
> It's likely that the window is appearing offscreen so you can try setting 640x480 compatibility otherwise a trace is needed.

Yes I tried all the settings I can put (640x480 ode, 8 or 16 bit color, admin privileges) but they didn't help. Can you tell how to create a trace?

**Comment by cracyc** [comment]
https://github.com/otya128/winevdm/issues/226#issuecomment-779702542

**Comment by BEENNath58** [comment]
> [#226 (comment)](https://github.com/otya128/winevdm/issues/226#issuecomment-779702542)

Here is the trace:
[trace.txt](https://github.com/otya128/winevdm/files/9822970/trace.txt)

**Comment by cracyc** [comment]
From your trace:
```
0fa4:Call USER.452: CREATEWINDOWEX(00000000,13074f52 "Afx:2203:1266:5:1416",1307007b "Sierra On-Line Setup",01cc0000,0780,0000,0000,0438,0000,0000,1306,00000000) ret=12df:924d ds=1307
```
This means the window is appearing at 1920x0 with size 0x1280, so offscreen to the right with 0 height.

On my computer for comparison:
```
2578:Call USER.452: CREATEWINDOWEX(00000000,12f74f52 "Afx:2203:1256:5:140e",12f7007b "Sierra On-Line Setup",01cc0000,0000,0000,0dfe,05a0,0000,0000,12f7,00000000) ret=12cf:924d ds=12f7
```

**Comment by BEENNath58** [comment]
What is the solution/workaround then...?

**Comment by sastofficial** [comment]
Trying to install half-life (1999 release) and it doesnt work either (dont tell me to use steam i already have it on steam i wanna try the old version)

**Comment by cracyc** [comment]
Need a trace to be certain it's the same problem.

**Comment by Froakiein2022** [comment]
While I was trying to install CyberGladiators (which is a Sierra game), the installer plays sound and the Sierra icon displays on the taskbar, but the installer window itself doesn't display. 

Here's the trace:
[trace.txt](https://github.com/user-attachments/files/16036032/trace.txt)

**Comment by cracyc** [comment]
This is the same problem.  It doesn't happen for me so I can only guess.  Try uncommenting WINVER=3.1 in your otvdm.ini file.

**Comment by Froakiein2022** [comment]
> This is the same problem. It doesn't happen for me so I can only guess. Try uncommenting WINVER=3.1 in your otvdm.ini file.

Uncommenting WINVER=3.1 in your otvdm.ini file solved that problem. Thanks.
